### DISCUSSION TITLE
How to access postcss filter in JSX component?

### DISCUSSION THREAD

**Discussion by shishkin** [lumeland/lume#368] [Q&A] [answered]
Suppose I have a JSX (or preact JSX) component:

```tsx
export const css = `
p.test {
  color: red;
}
`;

export default ({}, {postcss}) => (<p className="test">Hello</p>);
```

How can I call postcss filter on the CSS? I understand that postcss is only available on the default export, not the css export.

**Comment by oscarotero** [comment, marked as answer]
Do you mean Lume components? (components stored in the `_components` folder). If so, the postcss plugin already process the css code because Lume create a css file with the CSS code generates by the used components (by default `/components.css`), and this file is processed as any other CSS file.

**Reply by shishkin** [reply]
Yes, that's perfect. And does the same apply for `esbuild` and `js` export of a lume component?

**Reply by oscarotero** [reply]
Yes. Lume first creates the files with the css and javascript code exported by the components and then run the processors (postcss, esbuild, lightningcss, etc).
